src/diff_printer.rs

```diff
@@ -1,66 +1,97 @@
 use difference::{ Changeset, Difference };
-use colored::Colorize;
+use colored::{Color, Colorize, ColoredString};
 use std::fmt::{ Formatter, Display, Error };
 
 pub struct DiffPrinter(pub Changeset);
 
-fn print_line_number<F, D: Display>(current_line: Option<usize>,
-        f: &mut Formatter, colorizer: &F) -> Result<Option<usize>, Error>
-    where F: Fn(&str) -> D,
-{
+fn print_line_number(
+    current_line: Option<usize>, 
+    f: &mut Formatter, 
+    colorizer: Colorizer
+) -> Result<Option<usize>, Error>  {
```

Comment:
Actually, can you revert these back onto 1 or 2 lines for the parameter and return type? Sorry, I'm just not the biggest fan of 1 parameter per line style